presto-tests/src/test/java/com/facebook/presto/tests/TestLocalQueries.java

```diff
@@ -144,4 +144,10 @@ public void testIOExplain()
                 jsonCodec(IOPlan.class).fromJson((String) getOnlyElement(result.getOnlyColumnAsSet())),
                 new IOPlan(ImmutableSet.of(input), Optional.empty()));
     }
+
+    @Test
```

Comment:
Move this test to `AbstractTestQueries`, which will be run in different configurations rather than just with `LocalQueryRunner`.